test/server/cards/04-MM/SirenHorn.spec.js

```diff
@@ -0,0 +1,28 @@
+describe('Siren Horn', function () {
+    integration(function () {
+        describe("Siren Horn's ability", function () {
+            beforeEach(function () {
+                this.setupTest({
+                    player1: {
+                        house: 'saurian',
+                        hand: ['siren-horn'],
+                        inPlay: ['senator-shrix']
+                    },
+                    player2: {
+                        inPlay: ['little-rapscal', 'troll', 'nexus']
+                    }
+                });
+            });
+
+            it('should make the fight target gain 1 amber and the upgraded creature lose 1 amber', function () {
```

Comment:
Could be useful to add a test case when Senator Shrix has no amber on it.

Reply:
Good idea!